iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -124,8 +124,10 @@ void buildFlowTransformPassPipeline(OpPassManager &passManager) {
   passManager.addNestedPass<FuncOp>(mlir::createCSEPass());
 
   // Legalize constants to be valid for IREE.
-  passManager.addNestedPass<FuncOp>(mlir::createCanonicalizerPass());
-  passManager.addNestedPass<FuncOp>(IREE::Flow::createPromoteI1ToI8Pass());
+  // TODO(suderman): Determine why adding these passes causes failures on
```

Comment:
I created an issue for it, we can put the issue ID here.

```suggestion
  // TODO(GH-6070): Determine why adding these passes causes failures on
```